grid_map_common/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>grid_map_common</name>
+  <version>0.1.0</version>
+  <description>Common support functionality used throughout grid_map</description>
+  <maintainer email="someone@gmail.com">who_is_the_maintainer</maintainer>
```

Comment:
This should be filled in. Also bump the  versions of all the packages to 2.0.0 so we have a clear versioning separation

Reply:
@SteveMacenski should I write you down as maintainer?

Reply:
Add me + the original maintainers on the ROS1 packages (and maybe you, if you want)

Reply:
Thanks, I will check with our DevOps.